### PR TITLE
fix colons

### PR DESCRIPTION
**Pull request by robbrockbank**
A bunch of missing colons.

### CODE REVIEW

File: master/reference/calicoctl/commands/node/run.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Options:
                            [default: /var/log/calico]
      --node-image=<DOCKER_IMAGE_NAME>
                            Docker image to use for Calico's per-node container.
-                           [default: quay.io/calico/node:%s]
+                           [default: quay.io/calico/node:latest]
```
Comment (ozdanborne) [inline master/reference/calicoctl/commands/node/run.md]: This should still be `{{site.data.versions[page.version].first.components["calico/node"].version}}`

File: master/reference/calicoctl/commands/node/run.md [hunk 2]
```diff
@@ -278,7 +278,7 @@ sudo calicoctl node run --ip autodetect --ip-autodetection-method interface=eth.
                          [default: /var/log/calico]
    --node-image=<DOCKER_IMAGE_NAME>
                          Docker image to use for Calico's per-node container.
-                         [default: quay.io/calico/node:%s]
+                         [default: quay.io/calico/node:latest]
```
Comment (ozdanborne) [inline master/reference/calicoctl/commands/node/run.md]: This should still be `{{site.data.versions[page.version].first.components["calico/node"].version}}`

File: v2.3/reference/calicoctl/commands/node/run.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ Options:
                            [default: /var/log/calico]
      --node-image=<DOCKER_IMAGE_NAME>
                            Docker image to use for Calico's per-node container.
-                           [default: quay.io/calico/node:%s]
+                           [default: quay.io/calico/node:latest]
```
Comment (ozdanborne) [inline v2.3/reference/calicoctl/commands/node/run.md]: `{{site.data.versions[page.version].first.components["calico/node"].version}}`

File: v2.3/reference/calicoctl/commands/node/run.md [hunk 4]
```diff
@@ -278,7 +278,7 @@ sudo calicoctl node run --ip autodetect --ip-autodetection-method interface=eth.
                          [default: /var/log/calico]
    --node-image=<DOCKER_IMAGE_NAME>
                          Docker image to use for Calico's per-node container.
-                         [default: quay.io/calico/node:%s]
+                         [default: quay.io/calico/node:latest]
```
Comment (ozdanborne) [inline v2.3/reference/calicoctl/commands/node/run.md]: `{{site.data.versions[page.version].first.components["calico/node"].version}}`

File: master/reference/calicoctl/commands/node/run.md (hunk 1 above)
Comment (robbrockbank) [inline master/reference/calicoctl/commands/node/run.md]: nope - this is hardcoded as latest now - because of the calico/node split.

Ditto below.

**Pull request merged**